src/test/java/com/ncc/aif/ScalingTest.java

```diff
@@ -70,7 +70,7 @@
  */
 public class ScalingTest {
 
-    private static final String LDC_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/LdcAnnotations#";
+    private static final String LDC_NS = "https://tac.nist.gov/tracks/SM-KBP/2019/LdcAnnotations#";
```

Comment:
This change is fine, but there needs to be a similar change in python/tests/ScalingTest.py on line 12

Reply:
Fixed